package.json

```diff
@@ -15,15 +14,17 @@
     "build:ssr": "yarn workspace ssr build",
     "clean": "rm -fr client/build && rm -fr ssr/dist && rm -fr content/_all-titles.json",
     "prebuild": "yarn build:client && yarn build:ssr && node content build --ensure-titles",
-    "start": "yarn run prebuild && nf start"
+    "start": "yarn run prebuild && nf start",
+    "functionalstart": "set -a && source testing/.env && yarn run prebuild && nf -e testing/.env start"
```

Comment:
Naming seems a lil mixed. The directory is called `testing`, this command `functional` and the name in package.json is `functionaltests`. I'd align that if I were you.
And more specific to this script `functional:start` would be maybe more in line with the other scripts.

`set -a` doesn't run on Windows. Maybe we can change how we call dotenv for build like:
```js
require("dotenv").config({path: process.env.NODE_ENV === 'testing' ? `testing/.env` : `.env`);
```
then again, that'd move knowledge about testing into all the places where we use dotenv. So maybe that's worse.

Reply:
+1 on the rename. 

Foreman (the `nf` executable) is the only thing that can't accept that you have environment variables set before. 
I.e. this works:
```js
// This is dummy.js
require("dotenv").config();
console.log('FOO==', process.env.FOO);
```
And it's respecting environment variables BEFORE reading the `.env`
```
cat .env
FOO=different
export FOO="bar"
node dummy.js
```
outputs:
```
FOO== different
```

Now, this is NOT true for foreman (`nf`). foreman will always read the `.env` file no matter what. And the only way to override that is with the `nf -e different.env.file start`.

I wonder if we can simply use `ENV_FILE=testing/.env` and everywhere where we use `require("dotenv").config({path: process.env.ENV_FILE || '.env'})`. 
That means we wouldn't need `set -a`. 
I'll try that.